django_project/core/base_templates/index.html

```diff
@@ -34,7 +34,24 @@
         <nav class="navbar">
             <div class='container'>
                 <p class="open-edu-logo"><span class='text-white'>Open</span><span class='text-blue'>Edu</span></p>
-                <a class='text-white' href='/site-admin'>Administration</a>
+            <div class="col-md-8">
+            {% if user.is_authenticated %}
+                <p class="text-white text-right">
+                    Welcome, <b>{{ user.username }}</b>
```

Comment:
remove the `welcome`

Reply:
yes, it's a sample only. do we have mockup for this part?
i imagine we will have a burger kind of menu contains, at the very least, logout option. what else do we need here? dashboard? link to admin page?

Reply:
No. but for now, the username + a logout link is ok.